tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera.py

```diff
@@ -25,6 +26,7 @@ def test_unlock_to_camera(self):
         lock_screen = LockScreen(self.marionette)
         lock_screen.switch_to_frame()
         camera = lock_screen.unlock_to_camera()
+        Wait(self.marionette).until(lambda m: self.device.is_locked == False)
```

Comment:
Agreed with @mwargers. Regarding the changes below, we'll also need to add them in `lock_screen.unlock()` and `passcode_pad.type_passcode()`